setup.py

```diff
@@ -3,10 +3,18 @@
 from glob import glob
 from os.path import basename
 from os.path import splitext
-try: # for pip >= 10
+
+from pip import __version__ as pip_version
+major_version = int(pip_version.split('.')[0])
+if major_version >= 20:
+    from pip._internal.req import parse_requirements
+    attr = 'requirement'
```

Comment:
could we change this to `requirement_attr` rather than `attr`. just so  we know the why of setting this earlier.